### ISSUE TITLE
Option to use the current block instead of creating another

### ISSUE THREAD

**Issue by freMea** [elvirbrk/NoteHighlight2016#185]
Hi,

First : awesome feature.

The only problem I have is I can really paste code on the flow of current page since it creates a new block each time.

I hope it could detect the context so it doesn’t need to do it when cursor is in a block already.

**Comment by mattlohkamp** [comment]
Agreed, inserting the formatted text in a block at the top of the current page is unexpected (and, in many cases, unwanted) behavior. Text should be inserted at the location of the caret in the current document.
